Makefile

```diff
@@ -449,9 +458,43 @@ endif
 # This provides a Makefile simulation of a Meta-internal folly integration.
 # It is not validated for general use.
 ifeq ($(USE_FOLLY),1)
```

Comment:
I'd like to see some kind of comment explaining USE_FOLLY vs. USE_FOLLY_LITE and FOLLY_DIR vs. FOLLY_PATH.